src/global.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
What do you think about setting the `box-sizing` property globally to the `border-box`?

Reply:
I don't know if it is a good idea. Mostly we are going to use MUI components and `box-sizing` set globally might override MUI components' styles.